themes/ddbasic/sass/components/menu.scss

```diff
@@ -0,0 +1,542 @@
+@import '../base.scss';
```

Comment:
Describe component contents/purpose, why are these styles grouped? What is the common denominator?